packages/dropdowns/src/elements/menu/Menu.tsx

```diff
@@ -87,12 +88,12 @@ export const Menu = forwardRef<HTMLUListElement, IMenuProps>(
       typeof button === 'function' ? (
         button(triggerProps)
       ) : (
-        <StyledButton {...triggerProps}>
+        <Button {...triggerProps} data-garden-id={BTN_COMPONENT_ID}>
           {button}
-          <StyledButton.EndIcon isRotated={isExpanded}>
+          <Button.EndIcon isRotated={isExpanded}>
             <ChevronIcon />
-          </StyledButton.EndIcon>
-        </StyledButton>
+          </Button.EndIcon>
+        </Button>
```

Comment:
Let's keep `StyledButton` restored rather than making architecture assertions in this PR.